ui/apps/platform/src/Containers/MainPage/InviteUsersModal.tsx

```diff
@@ -165,26 +230,92 @@ function InviteUsersModal(): ReactElement | null {
             hasNoBodyWrapper
         >
             <ModalBoxBody>
-                {/* <p>{JSON.stringify(authProvidersWithRules)}</p> */}
-                <InviteUsersForm
-                    formik={formik}
-                    providers={authProviders}
-                    roles={allowedRoles}
-                    onChange={onChange}
-                />
+                {authProviders.length === 0 && (
+                    <Alert
+                        title="No auth providers are available."
+                        variant="warning"
+                        isInline
+                        className="pf-u-mb-lg"
+                    >
+                        <Text>
+                            You must have at least one auth provider in order to invite users.
+                        </Text>
+                        <Text>
+                            Visit the{' '}
+                            <Link onClick={onClose} to={`${accessControlBasePath}/auth-providers`}>
+                                Access Control
+                            </Link>{' '}
+                            section to add an auth provider.
```

Comment:
An observation, not necessarily a change request. To apply your insight about internationalization of the phrase about the role, here is a rough draft:

To add an auth provider, visit: Link